src/main/webapp/WEB-INF/jsp/player/sheet/sheet-create.jsp

```diff
@@ -0,0 +1,37 @@
+<html>
+	<head>
+		<%@ page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1" isELIgnored="false" %>
```

Comment:
The charset should be UTF-8 (make sure all of your .jsp files have this)

Reply:
Done